src/erlcloud_cloudtrail.erl

```diff
@@ -115,6 +115,14 @@ describe_trails(Trails, Config) ->
     Json = [{<<"TrailNameList">>, [list_to_binary(T) || T <- Trails]}],
     ct_request("DescribeTrails", Json, Config).
 
+-spec describe_trails([string()], aws_config(), boolean()) -> ct_return().
+describe_trails([], Config, IncludeShadowTrails)
```

Comment:
lastly , first parameter is ignored here while it should not and this is a bug 
http://docs.aws.amazon.com/awscloudtrail/latest/APIReference/API_DescribeTrails.html

Reply:
It's not a bug. It's just limited API which is no worse than before. But I'll add the trail names support anyway.

Reply:
Speaking about bugs, erlcloud wasn't actually working with the `trailNameList` parameter correctly, and I fixed it.